etl/harmonize.py

```diff
@@ -0,0 +1,260 @@
+#
+#  harmonize.py
+#  etl
+#
+
+from pathlib import Path
+from typing import DefaultDict, Dict, List, Optional, Set
+from collections import defaultdict
+import json
+import cmd
+
+import click
+import pandas as pd
+from thefuzz import process
+
+from owid.catalog import Table, Dataset
+
+from etl.paths import REFERENCE_DATASET
+
+
+@click.command()
+@click.argument("data_file")
+@click.argument("column")
+@click.argument("output_file")
+def harmonize(data_file: str, column: str, output_file: str) -> None:
+    """
+    Given a data file in feather or CSV format, and the name of the columnn representing
+    country or region names, interactively generate a JSON mapping from the given names
+    to OWID's canonical names.
+
+    When a name is ambiguous, you can use:
+
+    n: to ignore and skip to the next one
+
+    s: to suggest matches; you can also enter a manual match here
+
+    q: to quit and save the partially complete mapping
+
+    If a mapping file already exists, it will resume where the mapping file left off.
+    """
+    t = read_table(data_file)
+    geo_column = t.reset_index()[column]
```

Comment:
```suggestion
    geo_column = t.reset_index()[column].astype("str")
```

Reply:
Did this plus a `dropna()` to avoid stringifying NaN and the like